src/Views/Login.vue

```diff
@@ -11,6 +11,19 @@ export default {
   name: 'LoginPage',
   components: {
     LoginForm
+  },
+  props: {
+    bounceto: {
+      type: String,
+      default: '/'
+    }
+  },
+  // this blindly assumes that having *an* auth token means a user is logged in.
+  // if the auth token isn't valid, it will log them out after the api returns 401.
+  created: function () {
+    if (localStorage.getItem('auth') !== null) {
```

Comment:
FYI localStorage operations in some browsers when in incognito/private will throw an exception on localstorage access. I'd avoid bloating this right now, and convert this to an issue to be resolved later.